math/src/helpers.rs

```diff
@@ -1,7 +1,19 @@
+use crate::field::{element::FieldElement, traits::IsTwoAdicField};
+
 pub fn next_power_of_two(n: u64) -> u64 {
     if n <= 1 {
         1
     } else {
         (u64::MAX >> (n - 1).leading_zeros()) + 1
     }
 }
+
+pub fn resize_to_next_power_of_two<F: IsTwoAdicField>(trace_colums: &mut [Vec<FieldElement<F>>]) {
+    trace_colums.iter_mut().for_each(|col| {
+        // TODO: Remove this unwrap. This may panic if the usize cant be
+        // casted into a u64.
```

Comment:
You have at least a few years of runway before this is a problem. `usize` to `u64` conversion can only fail if the machine word exceeds 64 bits. I don't know of any architectures for which that is true yet.